content/visualisation/visualisation_1.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Example Report Table"
+post = ""
+weight = 1
++++
+
+Below is an example of a fractional attribution report table, which shows the calculated ROAS by attribution channel.
+
+| conversionWindowStartDate | conversionWindowEndDate | channel           | conversions | revenue | spend  | roas |
+| ------------------------- | ----------------------- | ----------------- | ----------- | ------- | ------ | ---- |
+| 2022-07-03                | 2022-08-01              | Direct            | 702.5       | 138979.0| 10000.0| 13.9 |
+| 2022-07-03                | 2022-08-01              | Organic_Search    | 267.1       | 21685.1 | 10000.0| 2.2  |
+| 2022-07-03                | 2022-08-01              | Paid_Search_Other | 50.6        | 4624.5  | 10000.0| 0.5  |
+| 2022-07-03                | 2022-08-01              | Display_Other     | 21.3        | 2070.3  | 10000.0| 0.2  |
+| 2022-07-03                | 2022-08-01              | Referral          | 12.0        | 581.7   | 10000.0| 0.1  |
+| 2022-07-03                | 2022-08-01              | Unmatched_Channel | 4.1         | 544.7   | 10000.0| 0.1  |
+| 2022-07-03                | 2022-08-01              | Video             | 1.5         | 29.6    | 10000.0| 0.0  |
+
+This table shows the ROAS figures for each channel during the conversion window specified. It also shows the conversions attributed to each channel, and the revenue associated with that channel. In our example we have a uniform spend ($10000) for each channel but you can substitute and join this to your own internal dataset to accurately account for spend and ROAS for your own marketing budgets, see [Next steps](/accelerators/fractribution/next_steps/next_steps_1/).
```

Comment:
Is there also a way we could have this displayed graphically too? From a technical perspective it is all here, but these accelerators will also be used for sales / marketing activity too, so a graphical view would help here I believe.
If we could have some sort of line graph / bar graph to show Channel & ROAS and / or Channel & Conversions.

Reply:
I have added a couple of graphs - let me know if this is what you had in mind.

Reply:
Love it, just helps to show what can be achieved now. 
Can we adjust so that we have them showing in a browser page like we have on the Introduction page on the Web Analytics page ([here](https://docs.snowplow.io/accelerators/web/)) 

You can embed the image using the following URL:
https://www.figma.com/file/gCAqcElebVklji5lErT1IQ/macOS-Browser-UI-Kit-(Big-Sur-Update)-(Community)?node-id=0%3A1 
Save as a png then re-insert into the page. 

Also, if we can make the font of the visual title smaller, as it is currently bigger than any font on the accelerator.

Reply:
Ah that looks a lot nicer now! Thanks. I made the title smaller but can re-adjust if you have further suggestions.

Reply:
Looks fab, thank you :)

Reply:
On the above, I would suggest we add some `before` and `after` as well. This will make it really powerful.
`Conversions & roas before `and below/on the side the `conversions and roas after`.

_Suppose before is last-click attribution._

Reply:
Thanks for the suggestion @igneel64! I have calculated first and last touch attribution (as traditionally done), annoyingly they're not hugely different to the fractional attribution output - I have attached a screenshot comparing fractribution with traditional first and last touch attribution methods. Is this still a large enough difference to include, or could this just make it seem like last touch could be a 'good-enough' method? :P 
<img width="1430" alt="Screen Shot 2022-11-15 at 4 49 36 pm" src="https://user-images.githubusercontent.com/61371935/201836724-d4d8f414-4748-4bec-af5d-93d931bf48f7.png">

Reply:
Hi Zoe, what data is being used to generate the model? Is it synthetic data or has this come from a customer and been anonymised?

Reply:
@Craig-Mace I'll have to check with Mike when he's back from leave- he gave me the sample data. I know that it's not from our data generator though.

Reply:
This uses an anonymised dataset from the Google Merchandise Store (publicly available in BigQuery in `bigquery-public-data:google_analytics_sample.ga_sessions_*`)